doc/articles/how-uno-works.md

```diff
@@ -2,60 +2,110 @@
+#### Platforms Supported
 
-### Android
+Skia rendering is available on:
 
-On Android, all types that inherit from `Microsoft.UI.Xaml.FrameworkElement`, also inherit from the native [`ViewGroup` type](https://learn.microsoft.com/dotnet/api/android.views.viewgroup). That is to say, on Android, all XAML visual elements are also native views.
+- **macOS**
+- **Linux**
+- **Windows** (called just Desktop, as an alternative to *native* WinUI rendering)
+- **WebAssembly**
+- **Android**
+- **iOS**
 
-When rendered at runtime, certain `FrameworkElement` types implicitly create inner views that inherit from higher-level native view types. For example, `Image` implicitly creates an inner `NativeImageView`, where `NativeImageView` is an Uno-defined internal type that inherits directly from the native `Android.Widget.ImageView` type.
+#### How Skia Rendering Works
 
-### Skia (Linux with X11/Wayland, Linux Framebuffer, macOS, Windows)
+- The entire UI is drawn onto a Skia canvas.
+- There are **no native views**; all visuals are composed in Skia using vector graphics.
+- A minimal native shell (like a window or web canvas) hosts the Skia surface.
 
-On Linux, XAML visual elements are rendered directly to a [Skia](https://skia.org/) canvas. Unlike the other target platforms, there's no 'native view type' to speak of. The Skia canvas is hosted inside of a bare minimum X11 shell.
+> [!NOTE]
+> Because it bypasses native UI components, Skia can offer pixel-perfect rendering and visual consistency, but it may lack native platform behaviors such as accessibility or platform styling.
 
-## Uno.UI at build time
+You can find more details about using Skia rendering in your app [here](xref:Uno.Skia.Rendering).
 
-The codebase of an Uno Platform application is a mix of XAML markup, C# code, images, string resources, and miscellaneous other assets. At build time, Uno Platform takes code and creates a native application for the target being built, including binaries, properly named and packaged assets, etc. How does it happen?
+### Native Rendering Targets
+
+#### Web (WebAssembly)
+
+On WebAssembly, XAML elements are translated into HTML elements:
+
+- Panels and layout containers → `<div>`
+- Leaf controls like `TextBlock`, `Image` → semantic elements like `<p>`, `<img>`, etc.
+
+This rendering integrates with CSS and DOM APIs, enabling UI behavior consistent with web standards.
+
+#### iOS
+
+- All `FrameworkElement` types are backed by native `UIView` instances.
+- Controls like `Image` implicitly create native `UIImageView` subviews.
+- Native input, layout, and accessibility features are utilized.
+
+#### Android
+
+- All `FrameworkElement` types inherit from native `ViewGroup`.
+- Controls like `Image` wrap native `ImageView` instances.
+- Leverages Android’s native rendering, accessibility, and gesture systems.
```

Comment:
@morning4coffe-dev, @MartinZikmund, @jeromelaban seems to me this section would need more info, or at least an intro sentence after the "### Native Rendering Targets" header